draft-ietf-core-href.md

```diff
@@ -425,36 +425,41 @@ represented by using the `discard` alternative instead.
 The rules `scheme`, `authority`, `path`, `query`, `fragment`
 correspond to the (sub-)components of a CRI, as described in
 {{constraints}}, with the addition of the `discard` section.
-The `discard` section can be used when neither a scheme nor an
-authority is present.
-It then expresses path
-prefixes such as "/",
-"./", "../", "../../", etc.
+
+### The `discard` Section
```

Comment:
I think the bit of information that's still missing here is a concise explanation of *why* discard is needed at all.

Reply:
Hmm, it says »It then expresses the operations performed on a base CRI by CRI references that
are equivalent to URI references with relative paths and path prefixes such as "/", "./", "../", "../../", etc.«